.gitignore

```diff
@@ -33,6 +33,7 @@ venv/
 .vagrant/*
 .lock
 yarn-error.log
+core/tests/linter_tests/*
```

Comment:
We still want to keep this folder in source control in case the test suite expands.

Reply:
@kevinlee12 Ah, I had the wrong idea when I added that line (for some reason i thought that removed it from linting but would keep it in source control). Is there a way to specify that we don't want CircleCI to lint the files in core/tests/linter_tests/*, since some of these files intentionally have invalid syntax (for the sake of testing)?

Reply:
You can include it in `EXCLUDED_PATHS` in `pre_commit_linter`

Reply:
Ah, it seems like it's already been included there. Must have missed that when I merged. Thanks!